### PR TITLE
Update the XSOAR mirroring integration to be more Plug and Play

### PR DESCRIPTION
**Pull request by ronykoz**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Update the XSOAR mirroring integration to be more Plug and Play

### CODE REVIEW

File: Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.yml [hunk 1]
```diff
@@ -73,9 +73,21 @@ configuration:
...
+- display: Mirror to identical incident type
+  additionalinfo: Enable this option to mirror the incident to the exact same fields in this system(in case they exist).
+  name: mirror_identically
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Disable mirroring for incidents came from this integration
+  additionalinfo: Enable this option to disable mirroring of incidents came from the integration of XSOAR Mirroring. This adds `-sourceBrand:“XSOAR Mirroring”` to your query.
+  name: disable_from_same_integration
+  defaultvalue: ""
```
Comment (anara123) [inline Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.yml:85]:
```suggestion
  defaultvalue: "true"
```

File: Packs/XSOARmirroring/ReleaseNotes/2_0_0.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### XSOAR Mirroring
+- Maintenance and stability enhancements.
```
Comment (anara123) [inline Packs/XSOARmirroring/ReleaseNotes/2_0_0.md:4]: Lets elaborate. We removed few incident types, mapping, classifier.

**Review by anara123 — COMMENTED** [review]
lets have a quick demo

File: Packs/XSOARmirroring/ReleaseNotes/2_0_0.md (hunk 2 above)
Comment (ronykoz) [reply Packs/XSOARmirroring/ReleaseNotes/2_0_0.md:4]: Will go over it with @kirbles19

Comment (kirbles19) [reply Packs/XSOARmirroring/ReleaseNotes/2_0_0.md:4]: @anara123 What was removed exactly?

Comment (anara123) [reply Packs/XSOARmirroring/ReleaseNotes/2_0_0.md:4]: Removed:
- Incident field (frompong)
- Incident Types (Ping and Pong)
- Classifier
- Mapper out

All these are considered breaking changes.

**Review by anara123 — APPROVED** [review]

File: Packs/XSOARmirroring/ReleaseNotes/2_0_0.md (hunk 2 above)
Comment (kirbles19) [reply Packs/XSOARmirroring/ReleaseNotes/2_0_0.md:4]: But this is from the pack in general, not specific to the integration, right?
What should the RN for the integration itself be?

Comment (kirbles19) [reply Packs/XSOARmirroring/ReleaseNotes/2_0_0.md:4]: Also, I think users will want a bit more information about what exactly this breaks and what the workaround is.

**Pull request merged**